src/Redux/books/book.js

```diff
@@ -0,0 +1,32 @@
+import { data } from '../../Components/data';
+/* eslint-disable no-case-declarations */
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const initialState = data;
+
+const addBook = (newItem) => ({
+  type: ADD_BOOK,
+  payload: newItem,
+});
+
+const removeBook = (newItem) => ({
+  type: REMOVE_BOOK,
+  payload: newItem,
+});
+
+const booksReducer = (state = initialState, action) => {
```

Comment:
- [ ] As demonstrated in the project guidelines, this reducer should be exported as **default**. Kindly do that. Also fix the resulting import issue in the `configureStore.js` file for your code to work.

Reply:
Thank You for the review, Please I'd like you to explain what difference this will make. Why cant I export it as a named export?

Reply:
@mckent05 Again this is a listed requirement in the guidelines.